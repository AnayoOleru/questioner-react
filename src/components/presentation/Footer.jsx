import React, { Fragment } from 'react';

const Footer = () => (
  <Fragment>
    <footer className='Footer'>
      <p>Questioner. &copy; 2018</p>
    </footer>
  </Fragment>
);

export default Footer;
