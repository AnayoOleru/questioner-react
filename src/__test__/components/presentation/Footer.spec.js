import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../../components/presentation/Footer';

describe('Footer Component', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(<Footer />);
  });
  test('Footer component should render', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
