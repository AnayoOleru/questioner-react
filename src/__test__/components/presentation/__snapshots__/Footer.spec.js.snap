// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component Footer component should render 1`] = `ShallowWrapper {}`;
